differential-expression/README.md

```diff
@@ -1,25 +1,89 @@
+
+These requirements can be installed by following the instructions at the links above.
+The example R Notebooks are designed to check if additional required packages are installed and will install them if they are not.
+
+#### RStudio
+
+We have prepared [a quick guide to RStudio](https://github.com/AlexsLemonade/training-modules/blob/master/intro_to_R_tidyverse/00-rstudio_guide.md) as part of our training content that you may find helpful if you're getting started with RStudio for the first time.
+
+Note that the first time you open RStudio, you should select a CRAN mirror.
+You can do so by clicking `Tools` > `Global Options` > `Packages` and selecting a CRAN mirror near you with the `Change` button.
+
+#### Interacting with R Notebooks
 
-[This notebook](https://github.com/AlexsLemonade/refinebio-examples/blob/master/differential-expression/gene_DE.Rmd)
-illustrates how to identify genes that are differentially expressed in a dataset
-using the `limma` package.
-It also creates a volcano plot and Q-Q plot in order to assess the performance
-of the linear model.    
+You can open an R Notebook by opening the `.Rmd` file in RStudio.
+Note that working with R Notebooks requires certain R packages, but RStudio should prompt you to download them the first time you open one.
+This will allow you to modify and run the R code chunks.
+Chunks that have already been included in an example can be run by clicking the green play button in the top right corner of the chunk or by using **Ctrl + Shift + Enter** (**Cmd + Shift + Enter** on a Mac).
+See [this guide using to R Notebooks](https://bookdown.org/yihui/rmarkdown/notebook.html#using-notebooks) for more information about inserting and executing code chunks.
 
-*Input:* GSE111111.tsv, GSE111111_metadata.tsv    
-*Output:* Q-Q plot, volcano plot, GSE111111limmaResults.tsv  
+#### Using your own data
+
+For all the examples in this module, the gene expression data and sample metadata are stored in a `data/` directory.
+If you'd like to adapt an example to include data you've obtained from [refine.bio](https://www.refine.bio/), we recommend placing the files in the `data/` directory and changing the filenames and paths in the notebooks to match these files.
+We suggest saving plots and results to `plots/` and `results/` directories, respectively, as these are automatically created by the notebooks if you move notebooks outside of the GitHub repository structure.
+We recommend using the [`limma` user guide](https://www.bioconductor.org/packages/devel/bioc/vignettes/limma/inst/doc/usersguide.pdf)
+to help you set up your model in a way that takes into account your experimental
+set up and hypotheses.
 
 ***
-### Differential Expression Analyses in GenePattern
+## Alternative differential expression analysis with GenePattern
+
+[GenePattern notebooks](http://genepattern-notebook.org/example-notebooks/)
```

Comment:
Okay - the initial rationale for recommending GenePattern _notebooks_ specifically was a reproducibility angle. If we envision GenePattern as a potential avenue for folks that have less programming experience, then I think we should consider reworking this to talk about and point to the web server modules (the original GenePattern offering). Here's the link to the quick start guide: http://software.broadinstitute.org/cancer/software/genepattern/quick-start

If we put some text around GenePattern versioning (see: http://software.broadinstitute.org/cancer/software/genepattern/concepts#_Version_Numbers), that satisfies the aforementioned reproducibility concern.